### PR TITLE
Fix typos discovered by codespell

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

**Review by Mamman06 — APPROVED** [review]

**Review by Mamman06 — APPROVED** [review]

**Comment by Z4nzu** [comment]
Hello @cclauss  I found that auto checks test gone failed. Please look into it and fix. 

Thanks

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@
 - Host to IP 
 - [Xerosploit](https://github.com/LionSec/xerosploit)
 - [RED HAWK (All In One Scanning)](https://github.com/Tuhinshubhra/RED_HAWK)
-- [ReconSpider(For All Scaning)](https://github.com/bhavsec/reconspider)
```
Comment (pabloescobar42) [inline README.md:66]: https://github.com/pabloescobar42/pabloescobar42/commit/88b5c0ef2dcc6cb9a4a3ead52068af205a185072

**Review by pabloescobar42 — APPROVED** [review]

File: .github/workflows/lint_python.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ jobs:
       - run: isort --check-only --profile black . || true
       - run: pip install -r requirement.txt || pip install -r requirements.txt || pip install --editable .  || pip install . || true
       - run: mkdir --parents --verbose .mypy_cache
-      - run: mypy --ignore-missing-imports --install-types --non-interactive .
```
Comment (pabloescobar42) [inline .github/workflows/lint_python.yml:22]: AEAA 5+num

Comment (pabloescobar42) [inline .github/workflows/lint_python.yml:22]: - [ ] AEAA 5+num
#

**Review by pabloescobar42 — APPROVED** [review]

**Comment by cclauss** [comment]
@Z4nzu Status on this one?

**Review by Z4nzu — APPROVED** [review]

**Pull request merged**